integration_test/cloudbuild-integration-test.yaml

```diff
@@ -247,7 +261,7 @@
   args:
   - diff
   - 'integration_test/downloaded.dcm'
-  - 'integration_test/data/example-redacted.dcm'
+  - 'integration_test/data/example-redacted-jp2k.dcm'
```

Comment:
Can you delete the old no longer used files?

Reply:
Ok